cherab/solps/formats/balance.py

```diff
@@ -0,0 +1,165 @@
+
+# Copyright 2016-2018 Euratom
+# Copyright 2016-2018 United Kingdom Atomic Energy Authority
+# Copyright 2016-2018 Centro de Investigaciones Energéticas, Medioambientales y Tecnológicas
+#
+# Licensed under the EUPL, Version 1.1 or – as soon they will be approved by the
+# European Commission - subsequent versions of the EUPL (the "Licence");
+# You may not use this work except in compliance with the Licence.
+# You may obtain a copy of the Licence at:
+#
+# https://joinup.ec.europa.eu/software/page/eupl5
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the Licence is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the Licence for the specific language governing permissions and limitations
+# under the Licence.
+
+import os
+import copy
+import numpy as np
+from scipy.io import netcdf
+from scipy.constants import elementary_charge
+from raysect.core.math import Discrete2DMesh
+
+from cherab.core.math.mappers import AxisymmetricMapper
+from cherab.core.atomic.elements import hydrogen, deuterium, helium, beryllium, carbon, nitrogen, oxygen, neon, \
+    argon, krypton, xenon
+
+from cherab.solps.eirene import Eirene
+from cherab.solps.mesh_geometry import SOLPSMesh
+from cherab.solps.solps_plasma import SOLPSSimulation
+
+Q = elementary_charge
+
+# key is nuclear charge Z and atomic mass AMU
+_popular_species = {
+    (1, 2): deuterium,
+    (6, 12.0): carbon,
+    (2, 4.003): helium,
+    (7, 14.0): nitrogen,
+    (10, 20.180): neon,
+    (18, 39.948): argon,
+    (18, 40.0): argon,
+    (36, 83.798): krypton,
+    (54, 131.293): xenon
+}
+
+
+# Code developed by J. Harrison 9/4/2019
+def load_solps_from_balance(balance_filename):
+    """
+    Load a SOLPS simulation from SOLPS balance.nc output files.
+
+    """
+
+    # Open the file
+    fhandle = netcdf.netcdf_file(balance_filename,'r')
```

Comment:
Per [PEP-8](https://www.python.org/dev/peps/pep-0008/), please add a space after the comma. This applies to quite a few places throughout the file. Running `pylint balance.py` will point out the others.